packages/httpd/mod_ssl-smoke/runtest.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# vim: dict=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/httpd/Sanity/mod_ssl-smoke
+#   Description: try to reinstall mod_ssl and run httpd with mod_ssl
+#   Author: Ondrej Ptak <optak@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2014 Red Hat, Inc.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/bin/rhts-environment.sh || exit 1
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+PACKAGES=${PACKAGES:-"httpd"}
+FIPS=${FIPS:-"false"}
+PYTHON=${PYTHON:-python}
+if [[ -x /usr/bin/python ]]; then
+        PYTHON=/usr/bin/python
+elif [[ -x /usr/bin/python2 ]]; then
+        PYTHON=/usr/bin/python2
+elif [[ -x /usr/bin/python3 ]]; then
+        PYTHON=/usr/bin/python3
+elif [[ -x /usr/libexec/platform-python ]]; then
+        PYTHON=/usr/libexec/platform-python
+fi
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [[ $FIPS == "true" ]]; then
+	    if [ -e "/etc/system-fips" ] && grep -q 1 /proc/sys/crypto/fips_enabled; then
+                rlPass "fips mode enabled"
+	    else
+		rlFail "fips mode disabled. Test requires fips mode!"
```

Comment:
Hi, should this be skipping the test instead of failing? Asking because the line below specifically says "Skipping the test" and the command doesn't match that.

Reply:
@veruu 
Thanks for the review  and totally agree with your comment.
I update the code , please re-review again.

Reply:
Thanks for the update! Just a question since I'm not familiar with this type of test - does the `rhts-report-result` assure the rest of `rhPhase`s don't run? I'm just confused about mixing those two together and what effects it has

Reply:
Thanks for the review.

I  trigger to make "skipping"  by adding <params "FIPS" values="true" ...>  .
 rhts-report-result  reports "Skipped" and the recipe existed without issue/effect.
this  is "skipped" beaker job:# 3369892
and 
"passed" beaker job:# 3369899
 
*added missing "exit" after  rhts-report-result .

Reply:
Thanks for the changes! I'll squash the commits into one.